collaborative/IOU/heat_index.ipynb

```diff
@@ -162,218 +165,426 @@
+    "Next, load the data into memory. Because we are loading the entire dataset, this will take about 5 minutes, hang tight!"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "011a8667-243d-4767-85a4-f02235aada43",
-   "metadata": {
-    "tags": []
-   },
+   "id": "77dd5ea4-511a-4544-afee-896820f81a89",
+   "metadata": {},
    "outputs": [],
    "source": [
-    "month_subset = [5, 6, 7, 8, 9] # May, June, July, August, September\n",
-    "heatidx_hist_hour = heatidx_hist_hour.isel(time = heatidx_hist_hour.time.dt.month.isin(month_subset))"
+    "heatidx_hist_hour = ck.load(heatidx_hist_hour)"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "dc1779c5-78b0-4d80-8b08-008268d1611f",
+   "id": "c7a632f0-78ca-484b-86eb-b133c0a955e7",
    "metadata": {},
    "source": [
-    "Now, let's load the hourly summer heat index data into memory. Since we are retrieving 30 years worth of hourly summer data, this will take a few minutes (approx. 4 min)!"
+    "### 1b) Retrieve the projected Heat Index data\n",
+    "Next we will repeat the same data retrieval as we did for the historical data so we can compare. Like the historical data, retrieving the hourly projections data will also take a few minutes - hang tight!"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "0f513d81-4563-48ba-b66d-da8799941a7c",
+   "id": "4a3d55fd-304f-482b-a9c7-04e7859b5db7",
    "metadata": {},
    "outputs": [],
    "source": [
-    "heatidx_hist_hour = ck.load(heatidx_hist_hour)"
+    "# select future data\n",
+    "selections.data_type = 'Gridded'\n",
+    "selections.area_subset = 'CA Electric Load Serving Entities (IOU & POU)'\n",
+    "selections.cached_area = ['Pacific Gas & Electric Company']\n",
+    "selections.timescale = 'hourly'\n",
+    "selections.variable_type='Derived Index'\n",
+    "selections.variable='NOAA Heat Index'\n",
+    "selections.resolution = '9 km'\n",
+    "selections.scenario_historical = []\n",
+    "selections.scenario_ssp = ['SSP 3-7.0 -- Business as Usual']\n",
+    "selections.time_slice = (2040, 2070) # mid-century"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c3645ba5-3533-4cda-95ff-801f9f709d5e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "heatidx_proj_hour = selections.retrieve()\n",
+    "heatidx_proj_hour = get_closest_gridcell(heatidx_proj_hour, stn_lat, stn_lon)"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "38c7eee8-510d-4166-82f2-9a0016efb15d",
+   "id": "ae6ef35a-85a0-4cb7-9aff-012f2d442edb",
    "metadata": {},
    "source": [
-    "This data object `heatidx_hist_hour` now represents hourly data throughout the summer. \n",
+    "Now, we will load in the hourly heat index projections. Because we are loading the entire dataset, this will take a few minutes. Hang tight!"
```

Comment:
Maybe change to "Because we are loading and computing on the entire dataset, this will take a few minutes", just because the next text of "Processing data to read 4.14 MB of data into memory..." makes the compute/load seem much less significant than it actually is in the back-end.